MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/adapter/ExampleAdapter.java

```diff
@@ -68,24 +69,26 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
       ExampleItemModel detailItem = dataSource.get(position);
       ViewHolder viewHolder = (ViewHolder) holder;
 
-      String imageUrl = context.getString(detailItem.getImageUrl());
-
-      if (!imageUrl.isEmpty()) {
-        Picasso.with(context)
-          .load(imageUrl)
-          .into(viewHolder.imageView);
-      } else {
-        viewHolder.imageView.setImageDrawable(null);
-      }
-
-      if (detailItem.getShowNewIcon()) {
-        viewHolder.newIconImageView.setImageDrawable(ContextCompat.getDrawable(context, R.drawable.new_icon));
-      } else {
-        viewHolder.newIconImageView.setImageDrawable(null);
+      if (Build.VERSION.SDK_INT >= detailItem.getMinSdkVersion()) {
```

Comment:
How about instead of wrapping up all the code into a `if (Build.VERSION.SDK_INT >= detailItem.getMinSdkVersion())`, you do a `if (Build.VERSION.SDK_INT < detailItem.getMinSdkVersion()) { return; }` and the same thing with the previous `if` so that the code has two level less of indentation?

Reply:
Ok, `onBindViewHolder()` now reads as 

```java
  @Override
  public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {

    ExampleItemModel detailItem = dataSource.get(position);
    ViewHolder viewHolder = (ViewHolder) holder;

    if (holder.getItemViewType() == 0) {
      return;
    }

    if (Build.VERSION.SDK_INT < detailItem.getMinSdkVersion()) {
      return;
    }

    if (Build.VERSION.SDK_INT >= detailItem.getMinSdkVersion()) {
      String imageUrl = context.getString(detailItem.getImageUrl());

      if (!imageUrl.isEmpty()) {
        Picasso.with(context)
          .load(imageUrl)
          .into(viewHolder.imageView);
      } else {
        viewHolder.imageView.setImageDrawable(null);
      }

      if (detailItem.getShowNewIcon()) {
        viewHolder.newIconImageView.setImageDrawable(ContextCompat.getDrawable(context, R.drawable.new_icon));
      } else {
        viewHolder.newIconImageView.setImageDrawable(null);
      }

      viewHolder.titleTextView.setText(context.getString(detailItem.getTitle()));
      viewHolder.descriptionTextView.setText(context.getString(detailItem.getDescription()));
    }
  }
```